src/test/resources/application.properties

```
@@ -3,3 +3,6 @@ spring.datasource.driverClassName=org.h2.Driver
 spring.datasource.username=sa
 spring.datasource.password=password
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+
+jwt.expiration = 1200000
+jwt.secretKey = fkaafa1r88)_402492flsfal2=040vfs104jfjlf1804180
```

just leave them empty
```suggestion
jwt.expiration=
jwt.secretKey=
```